### PR TITLE
feat/personalizaion

### PR DESCRIPTION
**Pull request by new5558**

### CODE REVIEW

File: db_model.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+Base = declarative_base()
+
+user = os.environ.get("POSTGRES_USER")
+password = os.environ.get("POSTGRES_PASSWORD")
+host = os.environ.get("POSTGRES_HOST")
+db = os.environ.get("POSTGRES_DB")
+
+engine = create_engine("postgresql://" + user + ":" + password + "@" + host + "/" + db)
+print('connected to sql')
```
Comment (boompttn) [inline db_model.py:27]: if not important, remove print

File: db_model.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+class Event(Base):
+    __tablename__ = "event"
+
+    def __str__(self):
+        return str(self.__class__) + ": " + str(self.__dict__)
+
+    id = Column(Integer, primary_key=True)
+    organization_id = Column(Integer, ForeignKey("organization.id"))
+    location_id = Column(Integer, nullable=True)
```
Comment (boompttn) [inline db_model.py]: Add foreign key

File: main.py [hunk 3]
```diff
@@ -20,6 +84,7 @@ def serve():
     )
     server.add_insecure_port("[::]:" + port)
     server.start()
+    print('service started')
```
Comment (boompttn) [inline main.py:86]: remove print?

**Review by boompttn — APPROVED** [review]
LGTM

File: helper.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    temp = wrapper.StringValue()
+    temp.value = value
+    return temp
+
+def getTimeStamp(data):
+    if data is None:
+        return None
+    timestamp = Timestamp()
+    timestamp.FromDatetime(data)
+    return timestamp
```
Comment (boompikachu) [inline helper.py:24]: If not used, remove

File: db_model.py (hunk 2 above)
Comment (boompttn) [reply db_model.py]: not that important

File: db_model.py (hunk 1 above)
Comment (new5558) [reply db_model.py:27]: It is for logging.

File: main.py (hunk 3 above)
Comment (new5558) [reply main.py:86]: for logging.

File: db_model.py (hunk 2 above)
Comment (new5558) [reply db_model.py]: I copy from your code. What should I do?

Comment (new5558) [reply db_model.py]: https://github.com/hu-tao-supremacy/participant/blob/1d73155b2422509e499b6d3ab64ba6ec22ab449b/db_model.py#L33

Comment (new5558) [reply db_model.py]: FIxed, just make sure to fix your code.

**Review by boompttn — APPROVED** [review]
LGTM

**Pull request merged**